pkg/pipelines/tekton/pipelines_pac_provider.go

```diff
@@ -154,7 +154,12 @@ func (pp *PipelinesProvider) createClusterPACResources(ctx context.Context, f fn
 		labels = pp.decorator.UpdateLabels(f, labels)
 	}
 
-	registry, err := docker.GetRegistry(f.Image)
+	img := f.Deploy.Image
```

Comment:
rationale: `f.Image` appears to be empty , `f.Deploy.Image` appears to contain the desired value instead.

Reply:
This caused error for all users not only admin!